### PR TITLE
set the storage size in the correct key

### PR DESCRIPTION
**Pull request by rberrelleza**
Set the storage size of the okteto registry in the correct key

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [x] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [x] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [x] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [x] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot)

### CODE REVIEW

File: okteto/config.yaml [hunk 1]
```diff
@@ -27,7 +27,9 @@ registry:
   storage:
     filesystem:
       enabled: true
-      size: 20Gi
+      persistence:
+        size: 20Gi
+        storageClass: civo-volume
```
Comment (rberrelleza) [inline okteto/config.yaml:32]: Is it ok to assume that this storage class is always available?

**Comment by saiyam1814** [comment]
The default is civo-volume so this can be ignored

**Comment by rberrelleza** [comment]
> The default is civo-volume so this can be ignored

this PR fixes another issue I discovered yesterday as well. I'll remove the volume and leave the other change.

**Comment by saiyam1814** [comment]
ok Merging

**Pull request merged**